### PR TITLE
Solar system

### PR DESCRIPTION
**Pull request by hanisntsolo**
A beautiful Solar System Animation using only html and css.

### CODE REVIEW

File: include.js [hunk 1]
```diff
@@ -292,6 +300,13 @@ let cards = [
     author: 'Simar',
     githubLink: 'https://github.com/SimardeepSingh-zsh'
   },
+  {
+    artName: 'solarsystem',
+    pageLink: './Art/Joy/solarsystem.html',
```
Comment (sophiabrandt) [inline include.js:305]: This should be `./Art/solasystem/solarsystem.html`

File: include.js [hunk 2]
```diff
@@ -292,6 +300,13 @@ let cards = [
...
+    imageLink: './Art/Joy/solarsystem.gif',
```
Comment (sophiabrandt) [inline include.js:306]: Same here, your image is not in the `Joy` folder, but in the `solarsystem` folder.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Please fix the links to your solar system artwork.

**Comment by hanisntsolo** [comment]
hi @sophiabrandt changes done.pls check

File: include.js (hunk 1 above)
Comment (hanisntsolo) [reply include.js]: @sophiabrandt done please check!

**Comment by hanisntsolo** [comment]
hi @sophiabrandt Can you please remove the changes reqeusted tag to this PR

**Comment by sophiabrandt** [comment]
It doesn't make any difference, but I can remove it.

**Pull request merged**